### ISSUE TITLE
Cannot load such file error

### ISSUE THREAD

**Issue by aptalca** [thekuwayama/nslookupot#4]
Hi I tried to install this in numerous ways and keep getting the same error when I run it:
```
Traceback (most recent call last):
        7: from /usr/bin/nslookupot:23:in `<main>'
        6: from /usr/bin/nslookupot:23:in `load'
        5: from /usr/lib/ruby/gems/2.6.0/gems/nslookupot-0.0.3/exe/nslookupot:6:in `<top (required)>'
        4: from /usr/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        3: from /usr/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        2: from /usr/lib/ruby/gems/2.6.0/gems/nslookupot-0.0.3/lib/nslookupot.rb:3:in `<top (required)>'
        1: from /usr/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
/usr/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require': cannot load such file -- caa_rr_patch (LoadError)
```

I tried:
1- Ruby 2.7 built from source on ubuntu bionic
2- Ruby 2.7 snap on ubuntu bionic
3- Ruby 2.6.5 snap on ubuntu bionic
4- Ruby 2.6.5 on Alpine 3.11 in docker

Am I doing something wrong?

Thanks

EDIT:
Installing version 0.0.2 works

**Comment by thekuwayama** [comment]
Thank you for comments!

You can resolve this error, installed `caa_rr_patch` gem.
To resolve this problem, I wrote the PR(#5). And I released [nslookupot-0.0.4](https://rubygems.org/gems/nslookupot/versions/0.0.4).

**Comment by aptalca** [comment]
Can confirm version 0.0.4 works.

Thanks so much.
